speech/cloud-client/src/main/java/com/example/speech/Recognize.java

```diff
@@ -186,6 +189,11 @@ public static void asyncRecognizeFile(String fileName) throws Exception, IOExcep
       List<SpeechRecognitionAlternative> alternatives = result.getAlternativesList();
       for (SpeechRecognitionAlternative alternative: alternatives) {
         System.out.printf("Transcription: %s%n", alternative.getTranscript());
+        for (WordInfo wordInfo: alternative.getWordsList()) {
+          System.out.println(wordInfo.getWord());
+          System.out.printf("\t%s ns - %s ns\n",
+              wordInfo.getStartTime().getNanos(), wordInfo.getEndTime().getNanos());
```

Comment:
Please update the other async sample to correctly show the seconds and nanos calculated to second fractions.